### DISCUSSION TITLE
Bug: Does Faust support Poylang?

### DISCUSSION THREAD

**Discussion by krishaamer** [wpengine/faustjs#1380] [Q&A]
## Applicable Versions

- `@faustjs/core` 0.2.9
- `@faustjs/react` 18.2.0
- `@faustjs/next` 13.2.4
- `WordPress version` 6.2

## Steps To Reproduce

1. Installed the example project: https://faustjs.org/docs/next/guides/project-walkthrough
2. Setup Faust plugin and Polyang in WP and created pages in 3 languages.
3. Created WP menus for each language
4. The same menu shows up in the front-end for all languages :(

Appreciate feedback on if it's *supposed* to work or are multiple languages currently not supported?

Thanks
Kris

**Comment by JEverhart383** [comment]
Hi @krishaamer you can use Faust with Polylang, but it might not offer the out-of-the-box support you are expecting. You still need to tell Faust which menus to load based on either the Next.js locale or the language of a particular post/page URI. I've got some experiments with Faust and Polylang in this repo that we're building for reference: https://github.com/JEverhart383/faust-intl

The wp-templates directory would be a good place to start, and if you're in Discord feel free to ask a question in community support

**Comment by theodesp** [comment]
Moving to conversation since this is a question and not a bug.
